src/features/filter.ts

```diff
@@ -1,10 +1,57 @@
-export const actions = { /* put action creators here */};
+import { Status } from '../types/Status';
 
-const filterReducer = () => {
+type SetStatus = {
+  type: 'filter/SET_STATUS',
```

Comment:
It would be better to extract 'filter/SET_STATUS' and 'filter/SET_QUERY' as constants and use them in your actions and reducer.